content/blog/2020-04-06-april-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,168 @@
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/april-20-heartbeat/347
+tags:
+  - Heartbeat
+  - Google Drive
+  - MLOps
+  - CI/CD
+  - Podcast
+  - DivOps
+---
+
+Welcome to the April Heartbeat, our
+[monthly roundup of cool happenings](https://dvc-landing-april-heart-6d0onb.herokuapp.com/tags/heartbeat),
+good reads and other bright spots in our community.
+
+## News
+
+**Adapting to the pandemic.** Although the world seems different than when we
+posted last month, the DVC community is steady and strong. As a predominantly
+distributed company, we've been developing our infrastructure for remote work
+from the get-go. It isn't always _easy_ to schedule an all-hands meeting across
+9 time zones but we make it work. This experience has prepared us well for the
+COVID-19 pandemic: although there are new challenges (like caring for families
+while working from home) we've been able to weather the transition to fully
+remote work relatively well.
+
+![](/static/uploads/images/2020-04-06/laptop_on_boat.jpeg)_Before social
+distancing started, DVC technical writer Jorge Orpinel Pérez has worked from a
+canoe. Check out more photos from his workations
+[on Instagram](https://www.instagram.com/workationer/)._
+
+**DVC sponsors DivOps.** In a time when many conferences are going remote out of
+necessity, we were fortunate to be part of an _intentionally_ remote conference
+this month! We sponsored [DivOps](https://divops.org/), a fully-online meeting
+led by women in DevOps. The DivOps lineup included speakers from GitHub,
+DropBox, Gremlin and more. DVC data scientist Elle (that's me!) gave a
+ten-minute talk about MLOps and CI/CD, so
+[please check out the video](https://dvc.org/blog/reimagining-devops-video).
+Another very relevant talk was from Anna Petrovicheva, CEO of
+[Xperience AI](http://xperience.ai/); Anna
+[spoke about her team's development workflow for deep learning projects](https://youtu.be/8nwpCQufeE0)
+and gave a clear overivew of how they use DVC.
+
+**DVC on the airwaves.** In early March, Elle was interviewed on an episode of
+[The Data Stream podcast](https://open.spotify.com/show/5w4sAKB0fT6lGCELZAMIBh)
+about a DVC data science project,
+[building a public dataset of posts](https://dvc.org/blog/a-public-reddit-dataset)
+from the "Am I the Asshole?" subreddit.
+
+<external-link
+href="https://open.spotify.com/episode/5JzIZLqnTF5aDh2B6UTemo?si=6LTQxq4xSDe0vhTpSLs1Jw"
+title="The Data Stream #3 - Who is the A-hole? With Elle"
+description="Ever wonder if it's possible to train a model to discover whether your friends are assholes or not? Today Elle comes on the show to talk about her project building a classifier to predict the results from reddit's hottest advice community: Am I the Asshole (or AITA for short)."
+link="spotify.com"
+image="/uploads/images/2020-04-06/data_stream.png"/>
+
+## New releases. This month, DVC has released some new features and updates:
```

Comment:
`This month, DVC has released some new features and updates:` looks a bit strange as part of the title

Reply:
also better to put a link to the the Github releases page ... since we mention only three items, which is not complete

Reply:
fixed!